src/argparse.cpp

```diff
@@ -82,6 +83,8 @@ void parseArguments(int argc, char **argv, std::vector<std::string>& slot_list,
                       << " DEBUG"
 #endif
                       << "\n";
+            if (!commit_hash.empty())
+               std::cout << "From source hash: " << commit_hash << "\n";
```

Comment:
It would be nice to also have it in the About box (and options too in fact).

Food for thought: Wouldn't it be better to concatenate it to VERSION_STRING instead (two versions from different commits are indeed different) ? It would be nice to have a way to remove it from the version in official releases though.